content/post/Start-up-dd.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Start-Up Deep Dive: MVP Building and User Generation "
+date: 2023-01-26T19:03:46+05:30
+authors: 
+    - nikita-ann-mathen
+Description: "A detailed summary of the fifth session of Startup Deep Dive on “MVP building” taken by our alumni."
+Tags: ["startup","startupdeepdive","MVP", "User Generation"]
+Categories: ["Startup"]
+DisableComments: false
+---
+
+A key ingredient to getting started on your entrepreneurial journey is finding the self-belief that it is possible for you to build an entire company from scratch. And what can be a more significant source of inspiration than hearing from alumni of your colleagues who have successfully achieved the same?
+
+
+Mr. Sandeep S and Ms. Ashwathy Venugopal are X-MECians and co-founders of Avasarshala, an ed-tech startup that curates talent-based opportunities for children. Ashwathy has even been recognized as one of the top 75 women entrepreneurs in India by the Central Government. Together, they joined us for the 5th session of IEDC MEC’s Entrepreneurial Training Series, Startup Deep Dive on  ‘MVP Building And User Generation’.
+
+
+So what is MVP? A Minimum Viable Product is a low-budget version of a product with just enough features to be used by early customers. The feedback obtained from the MVP can then be used to guide the development of the final product.
+
+
+_The MVP of a wedding cake would be a cupcake!_
```

Comment:
Can we try converting this to a quote instead using the \> operator like the following

>  "The MVP of a wedding cake would be a cupcake!"

Is this better? Will stand out more in a better way no?

Reply:
It was given in the document as Italics so I gave it like that, will change it and commit

Reply:
![image](https://user-images.githubusercontent.com/93091455/215074457-51b6b0b9-cb8e-4701-9045-a2335b7d854a.png)
is this fine?

Reply:
Sure let's go with this. 

Ig since this the quote is being directly referenced in the line after it, maybe it's better to not have a block-quote.
LGTM